provider/aws/config_test.go

```diff
@@ -17,59 +17,186 @@ limitations under the License.
 package aws
 
 import (
-	"context"
-	"os"
-	"testing"
+  "context"
```

Comment:
this formmatting does not looks right to me

Reply:
yup will fix that.. my IDE got messed up with editorconfig since the other project that I generally work on uses 4 spaces